src/core/decode_stream.js

```diff
@@ -73,7 +73,7 @@ class DecodeStream extends BaseStream {
     return this.buffer[this.pos++];
   }
 
-  getBytes(length, ignoreColorSpace = false) {
+  getBytes(length, decoderOptions = null) {
```

Comment:
It looks like setting this to `null` in the latest patch is causing the two mentioned regressions. Checking out this PR and opening that document shows this in the console: `Warning: Unable to decode image "img_p3_1": "TypeError: (destructured parameter) is null"`.